tasks/coverage/snapshots/semantic_babel.snap

```diff
@@ -2279,6 +2279,16 @@ Unresolved references mismatch:
 after transform: ["C", "T", "f"]
 rebuilt        : ["C", "f"]
 
+tasks/coverage/babel/packages/babel-parser/test/fixtures/typescript/tsx/type-parameters/input.ts
+semantic error: Scope children mismatch:
+after transform: ScopeId(1): [ScopeId(2)]
+rebuilt        : ScopeId(1): []
```

Comment:
i think these are fine. transformer does not remove TS scopes yet. https://github.com/oxc-project/oxc/issues/7460